specification/ai/Azure.AI.Projects/red-teams/client.tsp

```diff
@@ -0,0 +1,3 @@
+import "@azure-tools/typespec-client-generator-core";
```

Comment:
Why do we have a separate client.tsp for red teams?

Reply:
added in error, I've removed it